bucketeer/build.gradle

```diff
@@ -61,57 +63,54 @@ android {
 
 dependencies {
   // gRPC
-  implementation "io.grpc:grpc-okhttp:$versions.GRPC"
-  implementation "io.grpc:grpc-protobuf-lite:$versions.GRPC"
-  implementation "io.grpc:grpc-stub:$versions.GRPC"
-  compileOnly "javax.annotation:javax.annotation-api:$versions.JAVAX_ANNOTATION"
+  implementation libs.grpc.okhttp
+  implementation libs.okhttp
+  implementation libs.grpc.protobufLite
+  implementation libs.grpc.stub
+  compileOnly libs.javaxAnnotation
   // This provides the protobuf.Duration for event.proto
-  compileOnly "com.google.api.grpc:proto-google-common-protos:$versions.COMMON_PROTOS"
-  implementation "com.squareup.okhttp:okhttp:2.7.5"
-  implementation "com.google.protobuf:protobuf-javalite:$versions.PROTO_JAVALITE"
-
-  // Kotlin
-  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$versions.KOTLIN"
```

Comment:
Since Kotlin 1.4.0, we no longer need to declare a dependency on the stdlib library.


https://kotlinlang.org/docs/whatsnew14.html#dependency-on-the-standard-library-added-by-default